Simulator/player.py

```diff
@@ -1607,6 +1620,7 @@ def reset_state(self):
         self.health = 100
         self.max_units = 1
         self.num_units_in_play = 0
+        self.is_afk = False
 
     def spill_reward(self, damage):
```

Comment:
I think this method should only be handling the reward. If this is handling gold and augments and kayn and whatever else, there is an issue here. This could use a bit of a rework to make sure this other behavior is in the start of turn locations.